packages/components/locales/mi.json

```diff
@@ -49,10 +49,6 @@
     "description" : "Main title for page",
     "message" : "You can't view this page"
   },
-  "kzErrorPage.404.message" : {
```

Comment:
Hmm... I think this would've been related to the other PR. It's probably safe to actually update the message with `find` instead of having it deleted (I know it's auto generated, but maybe this can save us some cost from getting new translations? 🤞🏻)

Reply:
@HeartSquared yes, the previous PR got merged without a translation extraction. 

I've added it back and just replaced fing with find. For our translations we usually delete our old text so that the translator knows that it needs to be re-translated.

Reply:
Ah I see! That's probably a good call then. Didn't really think about it since we don't do i18n that often 😬 You've convinced me!